region/maridia/inner-green.json

```diff
@@ -1326,6 +1402,21 @@
               ],
               "openEnd": 1
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit By Evir Projectile",
+                  "notable": false,
+                  "requires": [
+                    "Gravity",
+                    "canTrickyJump"
+                  ],
+                  "note": "With no movement items besides Gravity, walk off the platform, jump on the sand and barely enter the sand fall before landing back on the platform and quickly getting to the door."
```

Comment:
The setup I like is: walk off the platform and face right.  Turnaround spinjump and enter the sand fall briefly.  Get to door.

My next favourite was running off into the sandfall then quickly running to the door.

Reply:
It if very difficult but possible to get hit in transition with Hijump only.
From the sand, turnaround spinjump towards the right.  Turn towards the sandfall about when you're level with the platform.  Turn right and shoot as soon as you enter the sandfall.  And try to land right next to the transition.

Reply:
Is this HiJump strat something you think should be added? I tried for a while and thought it wasn't possible

Reply:
It could be put in as an insane strat.  The only hard part is landing exactly next to the transition.